src/main/scala/com/karumi/katagenda/AgendaApplication.scala

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (C) 2015 Karumi.
```

Comment:
This should be our copyright since we are building this from scratch in a new lang and not just modifying their source